geode-docs/tools_modules/http_session_mgmt/chapter_overview.html.md.erb

```diff
@@ -45,6 +45,10 @@ These modules are included with the <%=vars.product_name_long%> product distribu
 
     Several log files are written by the various parts of the session management code.
 
+-   **[Configuring Non-Sticky Sessions](../../tools_modules/http_session_mgmt/configuring_non_sticky_sessions.html)**
+
+    This section describes the configuration of non-stick sessions.
```

Comment:
I think we should use "non-sticky" here, rather than "non-stick," for consistency. Or, if we prefer "non-stick" for some reason, we should change this term across throughout docs.